src/utils/data_loaders.py

```diff
@@ -44,6 +44,18 @@ def load_span_data(tokenizer_name, file_name, label_fn=None, has_labels=True):
     return list(rows.T.to_dict().values())
 
 
+def load_jsonl(data_file, tokenizer_name, max_seq_len, targ_map):
```

Comment:
can you give a docstring and maybe change the function name? This is a somewhat specific version of loading from JSONL.